### ISSUE TITLE
ValueError: Buffer dtype mismatch, expected 'float' but got 'double'

### ISSUE THREAD

**Issue by hainm** [kugupu/kugupu#8]
hi, I was trying the `5Visualisation.ipynb` notebook (for my curiosity) but getting below error. Any idea why?
I am sing centos-7 and I followed the "conda env create -f kgp_env.yml" steps.

```
ValueError                                Traceback (most recent call last)
<ipython-input-28-b4087e3e2557> in <module>
----> 1 v = kgp.visualise.draw_network(nets[0], show_molecules=True)
      2 v.add_unitcell()
      3 v

/scr2/external-programs/kugupu/kugupu/visualise.py in draw_network(network, view, color, show_molecules)
    235 
    236     # move contents of network into primary unit cell
--> 237     g = _gather_network(u.atoms.fragments)
    238 
    239     if view is None:

/scr2/external-programs/kugupu/kugupu/visualise.py in _gather_network(frags)
    139     for f in frags:
    140         print(f)
--> 141         mda.lib.mdamath.make_whole(f)
    142     # generate fragment positions
    143     pos = _gen_frag_positions(frags)

MDAnalysis/lib/_cutil.pyx in MDAnalysis.lib._cutil.make_whole()

ValueError: Buffer dtype mismatch, expected 'float' but got 'double'
```

**Comment by richardjgowers** [comment]
thanks @hainm , this is a bug we fixed recently, (I was using develop), I'm making a new release of mda to fix this....

**Comment by richardjgowers** [comment]
Ok this is fixed from MDA 0.20 onwards
